nidm/experiment/tests/test_experiment.py

```diff
@@ -84,6 +84,6 @@ def main(argv):
    main(sys.argv[1:])
 
 # very simple test, just checking if main doesnt give any error
-# def test_main():
-#     main(sys.argv[1:])
+def test_main():
```

Comment:
why is this PR touching the test_experiment file?

Reply:
@sanuann - you're working on quite old version, I've already uncommented this test and started skipping test_provone in #88. 
Yo should merge master first.